Database/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.1'
+
+services:
+  db:
+    container_name: plapy_prod
+    image: postgres
+    environment:
+      POSTGRES_USER: user
+      POSTGRES_PASSWORD: plapyDev
+      POSTGRES_DB: plapy_production_database
+    ports:
+      - "5432:5432"
```

Comment:
what are these ports used for? does this have to match the server port that is being used for the api or is it uncuppeled

Reply:
The first 5432 (before the colon) is the port on the host machine.
The second 5432 (after the colon) is the port inside the Docker container.
The reason you need both is to tell Docker: "Map port 5432 inside the container to port 5432 on the host."

Reply:
also quasi: das eine öfnnet es auf dem docker, das andere auf dem host(in dem fall localhost),so dass du im localhost:5432 direkt auf die databse kommst